_includes/proposals.html

```diff
@@ -1,33 +1,46 @@
 <article>
-  <h2 id="proposals" class="section-headline text-end">State of Proposals</h2>
+  <h2 id="proposals" class="section-headline text-end">
+    {{ site.data["site"]["navigation"]["state-of-proposals"] }}
+  </h2>
   <div class="featurelist">
-    <h3>Current Candidates for the Specification</h3>
+    <h3>
+      {{ site.data["proposals"]["title"] }}
+    </h3>
     <i class="featurelist__desc">
-      This section features proposals that are in Stage 3 of <a href="https://tc39.es/process-document/">our process</a>, which means they are close to completion.
+      {{ site.data["proposals"]["description"] }}
     </i>
     <details class="featurelist__legend">
-      <summary><strong>How to read the proposals list</strong></summary>
+      <summary><strong>
+          {{ site.data["proposals"]["how-to-read"]["title"] }}
+        </strong></summary>
       <p>
-      Each proposal has links out to other resources. The tags identify the following attributes.
-      Below you will find a table with example tags and their meanings.
-      All tags have hover text if you need help getting oriented.
+      {{ site.data["proposals"]["how-to-read"]["description"] }}
       </p>
 
-      <strong>Types of tags and what they mean</strong>
+      <strong>
+        {{ site.data["proposals"]["how-to-read"]["tags"]["title"] }}
+      </strong>
       <ul class="featurelist__legend__tags">
-          <li>Last Presented:
+          <li>
+            {{ site.data["proposals"]["how-to-read"]["tags"]["last-presented"] }}:
             <div class="featurelist__item__presented featurelist__item__tag">
-              <a title="Notes from most recent presentation">December 2018</a>
+              <a href="{{ presentation.url }}" title='{{ site.data["proposals"]["last-presented-alt"] }}'>{{ site.data["proposals"]["how-to-read"]["tags"]["last-presented-fake-date"] }}</a>
             </div>
           </li>
-          <li>Available Tests:
+          <li>
+            {{ site.data["proposals"]["how-to-read"]["tags"]["tests"] }}:
             <div class="featurelist__item__tests featurelist__item__tag">
-              <a title="Patch introducing tests in test262 repository">Test</a>
+              <a href="{{ url }}" title='{{ site.data["proposals"]["test-alt"] }}'>
```

Comment:
I'm not sure why, but the title is broken here (in English too)

Reply:
I will take a look. thanks!